src/styles/block/box.scss

```
@@ -0,0 +1,27 @@
+.box {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+
+  @media (max-width: 900px) {
+    & {
+      grid-template-columns: repeat(auto-fill, minmax($min-width-cell, 1fr));
+    }
```

It is already inside the selector.

```suggestion
    grid-template-columns: repeat(auto-fill, minmax($min-width-cell, 1fr));
```